index.d.ts

```diff
@@ -1,5 +1,14 @@
 declare namespace activeWindow {
 	interface Options {
+		/**
+		Enable the accessibility permission check. _(macOS)_
+
```

Comment:
This should mention that when disabled, it won't be able to get the URL of browsers.